container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -0,0 +1,62 @@
+package fluentbit
+
+const (
+	healthCheckEndpointPath = "api/v1/health"
+
+	fluentBitContainerName = "fluent-bit"
+	fluentBitImage         = "fluent/fluent-bit:latest-debug"
```

Comment:
using debug image for now with testing toolkit (`curl`, `netstat`) eventually will move to latest regular img